.env.example

```diff
@@ -9,3 +9,5 @@ SURVEY_SERVICE_DB_POOL_MIN=0
 SURVEY_SERVICE_DB_POOL_IDLE=10000
 SURVEY_SERVICE_LOGGING_LEVEL=emerg
 SURVEY_SERVICE_CORS_ORIGIN=http://localhost:4000
+JWT_SECRET=a-very-secret-secret
```

Comment:
@jmfadeley Use the test JWT we use for the other services (`0a6b944d-d2fb-46fc-a85e-0295c986cd9f`), or does it matter?

Reply:
So we don't use `SURVEY_SERVICE_CLIENT_SECRET` and the `.env.example` currently uses that test JWT. I think we should close this PR.